### PR TITLE
Add Script Commands: ping

### PR DESCRIPTION
**Pull request by crstauf**
`ping` and `ping-monitor`

### CODE REVIEW

**Review by unnamedd — APPROVED** [review]

File: developer-utils/ping.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# @raycast.schemaVersion 1
+# @raycast.title Ping address in clipboard
```
Comment (PitNikola) [inline developer-utils/ping.sh:4]: Let's name file `ping-address-from-clipboard`

File: developer-utils/ping.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# @raycast.mode fullOutput
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Ping an IP address or URL in clipboard.
+# @raycast.packageName Internet
+# @raycast.needsConfirmation true
```
Comment (PitNikola) [inline developer-utils/ping.sh:10]: Do you think this command worth having a confirmation? Doesn't seem to be that dangerous...

File: developer-utils/ping.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# @raycast.icon 🌐
+
+clipboard=$(pbpaste)
+ping -i 0.25 -t 3 "$clipboard"
```
Comment (PitNikola) [inline developer-utils/ping.sh:14]: How about using url validation here? See https://github.com/raycast/script-commands/blob/master/navigation/open-url-from-clipboard.sh

**Review by PitNikola — CHANGES_REQUESTED** [review]

File: developer-utils/ping.sh (hunk 2 above)
Comment (crstauf) [reply developer-utils/ping.sh:10]: Anytime I use the clipboard for specific content, I include a request for confirmation, as a double-check that clipboard contains valid content.

File: developer-utils/ping.sh (hunk 3 above)
Comment (crstauf) [reply developer-utils/ping.sh:14]: `ping` is not limited to URLs, can also be IP addresses.

File: developer-utils/ping.sh (hunk 1 above)
Comment (crstauf) [reply developer-utils/ping.sh]: Changed to `ping-from-clipboard.sh` to be inclusive of both URLs and IP addresses.

**Review by PitNikola — APPROVED** [review]

**Pull request merged**